README.md

```diff
@@ -734,6 +734,12 @@ const getTodoStandardWithMap = createStandardAction('GET_TODO').map(
 );
 ```
 
+### From `redux-actions`
```

Comment:
Could you please change to "Migrating from `redux-actions`"

Reply:
Done!